docker/flv2mp4-docker-image/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM centos:7
+
+MAINTAINER Hemani Katyal <hkatyal@redhat.com>
+
+RUN yum install epel-release -y && yum update -y && \ 
+    rpm --import http://li.nux.ro/download/nux/RPM-GPG-KEY-nux.ro && \
+    rpm -Uvh http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm && \
+    yum install ffmpeg ffmpeg-devel -y
+
+ADD flv2mp4.sh /usr/local/bin/
```

Comment:
I think more reproducible way (or easier to rebuild) would be to `wget` or `curl` this file from a location using the same tag as the image itself - `http://...../../.../1.0.0.Alpha16/flv2mp4.sh` or sth similar. WDYT @lordofthejars?

About the docker image itself, there are few questions we should address before releasing anything:

  * Should we keep docker file in the same repo as cube or create another one, similar to [jboss-dockerfiles](https://github.com/jboss-dockerfiles)?
  * How should we version/tag images? Should we follow versioning alongside Cube? So the first published image would be `1.0.0.Alpha16`